### PR TITLE
[PT FE] Add support for GFPGAN model

### PR DESCRIPTION
**Pull request by mvafin**
### Details:
 - *Add support for GFPGAN model*
 - *Support aten::normal, aten::normal_*

### Tickets:
 - *None*

### CODE REVIEW

**Review by pavel-esir — APPROVED** [review]
Looks great 🚀 👍 
Checked output images CPU, they are same as pytorch.

File: src/frontends/pytorch/src/op_table.cpp [hunk 1]
```diff
@@ -438,6 +439,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops_ts() {
         {"aten::new_zeros", op::translate_new_zeros},
         {"aten::nonzero", op::translate_nonzero},
         {"aten::norm", op::translate_norm},
+        {"aten::normal_", op::translate_normal_},
```
Comment (eaidova) [inline src/frontends/pytorch/src/op_table.cpp:442]: there is also normal without underscore
https://pytorch.org/docs/stable/generated/torch.normal.html#torch-normal
do we have plans to support it too?

Comment (mvafin) [reply src/frontends/pytorch/src/op_table.cpp:444]: Makes sense to add it here, even though it is a pretty much different op. Added

**Review by eaidova — APPROVED** [review]

File: tests/model_hub_tests/torch_tests/test_gfpgan.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                #  Cannot reliably validate the output 0
+                if i != 0:
+                    is_ok = False
+        assert is_ok, "Accuracy validation failed"
+
+    def teardown_class(self):
+        # remove all downloaded files from cache
+        self.repo_dir.cleanup()
+
+    @pytest.mark.precommit # todo: remove
```
Comment (mvafin) [inline tests/model_hub_tests/torch_tests/test_gfpgan.py:79]:
```suggestion
```

File: src/frontends/pytorch/src/op/rand.cpp [hunk 3]
```diff
@@ -25,7 +25,11 @@ namespace op {
 using namespace ov::op;
 
 namespace {
-OutputVector make_random_normal(const NodeContext& context, Output<Node> sizes, element::Type target_type) {
+OutputVector make_random_normal(const NodeContext& context,
+                                Output<Node> sizes,
+                                element::Type target_type,
+                                Output<Node> scale_const,
+                                Output<Node> mean_const) {
```
Comment (mvafin) [inline src/frontends/pytorch/src/op/rand.cpp:32]:
```suggestion
                                const Output<Node>& sizes,
                                element::Type target_type,
                                const Output<Node>& scale_const,
                                const Output<Node>& mean_const) {
```

**Pull request merged**